_posts/2022-11-07-road-to-quarkus-3.adoc

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: 'Starting Quarkus 3'
+date: 2022-11-07
+tags: announcement
+synopsis: 'Quarkus 3 is coming with new features and a move to Jakarta named APIs.'
+author: maxandersen
+---
+
+Quarkus 3 will continue the tradition of propelling Java development forward by moving fast and bring a cutting-edge user experience with the latest technologies.
+
+Happy to have a first Quarkus 3 Alpha available for users to try out and extension writers to integrate with to get early feedback.
```

Comment:
```suggestion
We are happy to make a first Quarkus 3 Alpha available for users to try out and extension writers to integrate with so that we get early feedback.
```